packages/react/src/ProgressBar/ProgressBar.tsx

```diff
@@ -7,16 +7,18 @@ import type {SxProp} from '../sx'
 import sx from '../sx'
 import {warning} from '../utils/warning'
 
-type ProgressProp = {progress?: string | number}
+type ProgressProp = {
+  progress?: string | number
```

Comment:
Should add `bg` as prop here as well. Bonus points if you type it based on `colors.*`

Reply:
Wouldn't this be a loooot fo colors? I am hoping we can soon get away from this and move to predefined values that map to css dsiplay colors.

Adding bg is a problem, because it is NOT optional for the item, but is optional for the progressBar. What to do? @siddharthkp

Reply:
> Wouldn't this be a loooot fo colors? 

Yes, but same as what we support now. I'm not sure what the exact types would look like, something like `typeof keyof colors`, i'd need to poke around to find out as well 😅

Reply:
Done, once you can give me the type I am happy to add it, or you just add a comment suggestions in the UI.